target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -160,7 +160,9 @@ ex2700)
 	;;
 f5d8235-v1|\
 f5d8235-v2)
+	ucidef_set_led_default "power" "POWER" "$board:blue:router" "1"
 	set_usb_led "$board:blue:storage"
+	set_wifi_led "$board:blue:wireless"
```

Comment:
The f5d8235-v2 has no blue wireless LED. Please split the f5d8235-v1 and f5d8235-v2 led config.